src/views/Common.tsx

```diff
@@ -0,0 +1,69 @@
+import { Suspense, useEffect } from 'react';
+import { lazily } from 'react-lazily';
+import { useAtom } from '@reatom/react';
+import { VisibleLogo } from '~components/KonturLogo/KonturLogo';
+import { AppFeature } from '~core/auth/types';
+import { urlStoreAtom } from '~core/url_store/atoms/urlStore';
+import { forceRun } from '~utils/atoms/forceRun';
+import { Row } from '~components/Layout/Layout';
+import { OriginalLogo } from '~components/KonturLogo/KonturLogo';
+import { currentRouteAtom } from '~core/router/atoms/currentRoute';
+import { userResourceAtom } from '~core/auth';
+import { Router } from '~core/router';
+import type { PropsWithChildren } from 'react';
+
+const { AppHeader } = lazily(() => import('@konturio/ui-kit'));
+const { NotificationToast } = lazily(() => import('~features/toasts'));
+const { PopupTooltip } = lazily(() => import('~features/tooltip'));
+const { SideBar } = lazily(() => import('~features/side_bar'));
+
+const DEFAULT_HEADER_TITLE = 'Disaster Ninja';
+
+export function Views() {
```

Comment:
didn't you say if file has single export it should be == to filename?

Reply:
nice catch!